R/stroke.R

```diff
@@ -141,18 +141,18 @@ best_link <- function(nodes, segments, links, edge_ids, flow_mode,
     return(linked_nodes)
   }
 
+  get_link_on_same_edge <- function(linked_segs, current_segment) {
```

Comment:
After some more thoughts on this, I  think this best suit being outside `find_best_link`, being on the same level as `get_linked_segments` and `get_linked_nodes`. Also, the variable we want to access without being a "real" input of the function is `edge_ids`, which is the vector storing all the original edge IDs for the segments.